runner/src/scenario/ceramic/new_streams.rs

```diff
@@ -163,6 +163,36 @@ pub async fn small_large_scenario(
         .register_transaction(instantiate_large_model))
 }
 
+pub async fn small_scenario(params: CeramicScenarioParameters) -> Result<Scenario, GooseError> {
+    let redis_cli = get_redis_client().await.unwrap();
+    let multiplexed_conn = redis_cli.get_multiplexed_tokio_connection().await.unwrap();
+    let shared_conn = Arc::new(Mutex::new(multiplexed_conn));
```

Comment:
i'm a little worried about putting an active connection in here for a long test.. i'm not sure how stable the redis client is and feel like I've seen weird dead connection issues but maybe i'm making that up. I'd probably prefer to just pull the redis client out of the user data and make a new connection unless the perf overhead is too much. we can try this first